requirements.txt

```diff
@@ -1,7 +1,7 @@
-PyPrint>=0.2.5,==0.2.*
-Pygments>=2.1.3
+PyPrint~=0.2.5
```

Comment:
because we're doing it for two projects and we switched to the complicated one because we thought it might help just to find out that it doesn't